include/haarfeature.h

```diff
@@ -0,0 +1,95 @@
+#ifndef INCLUDE_HAARFEATURE_H_
+#define INCLUDE_HAARFEATURE_H_
+#include <vector>
+
+namespace myHaar {
+
+enum haarFeatureType {HORIZONTAL_2x2, VERTICAL_2x2,
+                      HORIZONTAL_3x2, VERTICAL_3x2,
+                      HORIZONTAL_4x2, VERTICAL_4x2,
+                      FOURSQUARE};
+
+const haarFeatureType defaultType = HORIZONTAL_2x2;
+const int defaultWidth = 2;
+const int defaultHeight = 2;
+const int defaultScale = 1;
+const int defaultNumTypes = 7;
+const int defaultRatio3 = 3;
+const int defaultRatio4 = 4;
+
+class Point {
+ public:
+  explicit Point(int new_x = 0, int new_y = 0);
+  int getX() const;
+  int getY() const;
+  Point& operator=(const Point new_point);
+  ~Point() {}
+
+ private:
```

Comment:
H files should contain only public interface so please move out private section in implementation